docs/swagger.yaml

```diff
@@ -2177,6 +2189,18 @@ components:
           type: string
           example: "1212"
           description: "The team name."
+        invites:
+          type: array
+          items:
+            type: object
+          description: "The invites of the project"
+          example: [{"userIds": [40152855], "role": "copilot"}]
+        members:
+          type: array
+          items:
+            type: object
+          description: "The members of the project"
+          example: [{"lastName": "J", "role": "manager", "updatedBy": 8547899, "handle": "TonyJ", "userId": 8547899, "deletedBy": null, "createdAt": "2021-01-12T04:34:15.006Z", "firstName": "Tony", "createdBy": 8547899, "isPrimary": true, "id": 13819, "projectId": 16893, "email": "ajefts@topcoder.com", "updatedAt": "2021-01-12T04:34:15.018Z"}]
```

Comment:
Let's replace the real email `ajefts@topcoder.com` with some dummy one to avoid unnecessary exposure?